api/machine_types.go

```diff
@@ -379,6 +379,11 @@ type MachineServiceConcurrency struct {
 	SoftLimit int    `json:"soft_limit,omitempty" toml:"soft_limit,omitempty"`
 }
 
+type MachineFlyProxy struct {
+	AutostartMachine bool `json:"autostart_machine"`
+	AutostopMachine  bool `json:"autostop_machine"`
```

Comment:
same thing here, `*bool` is the only way we can know when a flag is not set and left to server defaults.